python/test/test_series.py

```diff
@@ -1,6 +1,6 @@
 from numpy import allclose, amax, arange, array, array_equal
```

Comment:
Any reason to split into two test modules? It's a `Data` method, so my first impulse would have been to put all these tests, with the full variety of keys, inside `test_data.py`. The conventions we use for integer keys on `Images` and tuple keys on `Series`, while common, are not strictly enforced, so seems a little weird to split the tests up based on that.